### PR TITLE
support configurable docker root

### PR DESCRIPTION
**Pull request by paynejacob**
Because users can change the docker root we need to support changing the container log path

### CODE REVIEW

File: packages/rancher-logging/rancher-logging.patch [hunk 1]
```diff
@@ -143,4 +143,6 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-logging/charts-original/values
 +
 +global:
 +  systemDefaultRegistry: ""
++
++dockerRoot: "/var/lib/docker/"
```
Comment (brendarearden) [inline packages/rancher-logging/rancher-logging.patch:147]: nit: a helper to ensure it ends with a slash incase a user forgets it

**Review by brendarearden — DISMISSED** [review]
lgtm

File: packages/rancher-logging/rancher-logging.patch (hunk 1 above)
Comment (paynejacob) [reply packages/rancher-logging/rancher-logging.patch:147]: That is a good idea

**Comment by paynejacob** [comment]
This is incomplete and most likely not compatible with recent chart changes.

**Pull request closed without merging**